kernel/kernel-api/src/main/java/io/delta/kernel/internal/metrics/TransactionMetrics.java

```diff
@@ -52,6 +52,7 @@ public void resetActionCounters() {
     addFilesSizeInBytesCounter.reset();
     removeFilesCounter.reset();
     totalActionsCounter.reset();
+    removeFilesSizeInBytesCounter.reset();
```

Comment:
Should we rename this method to `resetCounters` ? since `removeFilesSizeInBytesCounter` is not an action counter?